frontend/components/exercise/Badge.tsx

```diff
@@ -0,0 +1,34 @@
+const Badge = ({ number }) => {
```

Comment:
had already made CircledLabel. pretty sure that's what this is. can we use that please unless you notice there's something different